### ISSUE TITLE
[FEATURE REQ] Allow users to change QNN hyperparameters

### ISSUE THREAD

**Issue by SashwatAnagolum** [QForestCommunity/QForestML#3]
**Is your feature request related to a problem? Please describe.**
Currently the qnn hyperparameters are fixed: these include the bias, momentum, learning rate, batch size, etc. In order to make the model more flexible, we need to enable user manipulation of these values.

**Describe the solution you'd like**
New parameters added to the `trainNetwork` function, preferably keyword arguments, that allow users to change these values while making the API call to train the network.

**Describe alternatives you've considered (if any)**
None, but if anyone has another idea feedback would be appreciated.

**Additional context**
None.
